src/pages/Gateway/index.tsx

```diff
@@ -427,8 +428,10 @@ const Gateway = () => {
                 <StatsBox
                   title="Delegates"
                   value={
+                    // @ts-expect-error - delegates is currently available on the gateway
                     gateway?.delegates
-                      ? Object.keys(gateway.delegates).length
+                      ? // @ts-expect-error - delegates is currently available on the gateway
+                        Object.keys(gateway.delegates).length
```

Comment:
can remove these comments with new install of the ar.io sdk

Reply:
Can not; the new SDK removes delegates and adds an API for retrieving delegate data separately from the gateway. The delegates is still being returned as part of the gateway data structure, and getGatewayDelegates() needs a change in the alpha SDK. These comments will be used to identify places in the app where calls to get delegates will eventually be replaced.